bugbot/rules/inactive_patch_author.py

```diff
@@ -0,0 +1,211 @@
+# # This Source Code Form is subject to the terms of the Mozilla Public
+# # License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import re
+from typing import Dict, List
+
+from libmozdata.connection import Connection
+from libmozdata.phabricator import PhabricatorAPI
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+from bugbot import people, utils
+from bugbot.bzcleaner import BzCleaner
+from bugbot.nag_me import Nag
+from bugbot.user_activity import PHAB_CHUNK_SIZE, UserActivity, UserStatus
+
+logging.basicConfig(level=logging.DEBUG)
+PHAB_FILE_NAME_PAT = re.compile(r"phabricator-D([0-9]+)-url\.txt")
+
+
+class InactivePatchAuthors(BzCleaner, Nag):
+    """Bugs with patches authored by inactive patch authors"""
+
+    def __init__(self):
+        super(InactivePatchAuthors, self).__init__()
+        self.phab = PhabricatorAPI(utils.get_login_info()["phab_api_key"])
+        self.user_activity = UserActivity(include_fields=["nick"], phab=self.phab)
+        self.default_assignees = utils.get_default_assignees()
+        self.people = people.People.get_instance()
+        self.no_bugmail = True
+
+    def description(self):
+        return "Bugs with inactive patch authors"
+
+    def columns(self):
+        return ["id", "summary"]
+
+    def get_bugs(self, date="today", bug_ids=[], chunk_size=None):
+        bugs = super().get_bugs(date, bug_ids, chunk_size)
+
+        rev_ids = {rev_id for bug in bugs.values() for rev_id in bug["rev_ids"]}
+        try:
+            inactive_authors = self._get_inactive_patch_authors(list(rev_ids))
+        except Exception as e:
```

Comment:
I would avoid using `Exception` as the error-catching type where possible.

Reply:
Makes sense! I removed the try-except block here: 243515d9bc4eac610e39a5cba6da1319a98bee3b. I made sure to do a dry-run and noticed that I did not encounter any errors, so I don't think the exception handler is necessary.